pkg/roachprod/vm/startup.go

```diff
@@ -34,6 +34,10 @@ const (
 	EbpfExporterPort = 9435
 	// EbpfExporterMetricsPath is the path that EbpfExporter serves metrics on.
 	EbpfExporterMetricsPath = "/metrics"
+	// CgroupExporterPort is the port that EbpfExporter listens on.
+	CgroupExporterPort = 9101
```

Comment:
This one is technically taken although I have no clue what `Bacula Director` is/does,

```
cat /etc/services |grep 9101
bacula-dir	9101/tcp			# Bacula Director
```